corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -269,14 +269,23 @@ <h3 data-bind="text: $root.activeCaseType()" style="display: inline-block;"></h3
               </div>
               {% if request|toggle_enabled:"CASE_IMPORT_DATA_DICTIONARY_VALIDATION" %}
                 <div class="row-item main-form">
-                <select class="form-control"
+                  <select class="form-control"
+                          data-bind="
+                            options: $root.availableDataTypes,
+                            optionsCaption: 'Select a data type',
+                            optionsText: 'display',
+                            optionsValue: 'value',
+                            value: dataType,
+                            disable: isGeoCaseProp,">
+                  </select>
+                  <span class="hq-help" style="height: fit-content"
                         data-bind="
-                                    options: $root.availableDataTypes,
-                                    optionsCaption: 'Select a data type',
-                                    optionsText: 'display',
-                                    optionsValue: 'value',
-                                    value: dataType,
-                                "></select>
+                          popover: {
+                            content: '{% blocktrans %}This GPS case property is currently being used to store the geolocation for cases, and so the data type cannot be changed.{% endblocktrans %}',
+                            trigger: 'hover' },
```

Comment:
nit:
"the geolocation for cases, so the"

Reply:
[09d3a67](https://github.com/dimagi/commcare-hq/pull/33661/commits/09d3a678460cde9042058b2d9c2d5687fd7eec83)